From a067e530be4d8837b6b73dc211ae19edbad725a4 Mon Sep 17 00:00:00 2001
From: Peter Yin <peter.yin@quantatw.com>
Date: Fri, 30 Aug 2024 15:45:27 +0800
Subject: [PATCH 1006/1008] ARM: dts: aspeed: Harma: revise i2c0 and i2c2 duty
 cycle

Revise the duty cycle for I2C0 and I2C2 to match the definitions in the datasheet.
This patch only in meta,
base on:https://github.com/facebook/openbmc/blob/helium/common/recipes-kernel/linux/6.6/0160-aspeed-i2c-add-clock-duty-cycle-property.patch

Signed-off-by: Peter Yin <peter.yin@quantatw.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
index 70a416c38dce..f822275802a2 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
@@ -203,6 +203,7 @@ &kcs3 {
 
 &i2c0 {
 	status = "okay";
+	i2c-clk-high-min-percent = <68>;
 
 	pwm@5e {
 		compatible = "maxim,max31790";
@@ -275,6 +276,7 @@ eeprom@50 {
 
 &i2c2 {
 	status = "okay";
+	i2c-clk-high-min-percent = <68>;
 
 	pwm@5e {
 		compatible = "maxim,max31790";
-- 
2.47.3

